source/includes/trigger-auto-resume.rst

```diff
@@ -0,0 +1,5 @@
+If enabled, when this Trigger's resume token
+cannot be found in the cluster's oplog, the Trigger automatically resumes
+processing events at the next relevant change stream event.
```

Comment:
What is "relevant" here? e.g. "oldest event on the oplog" or "new events after the trigger is resumed".

Edit - I guess it's the rest of the note that explains 😅